src/pages/en/guides/markdown-content.md

```diff
@@ -368,22 +375,120 @@ export default {
     render: [
       '@astrojs/markdown-remark',
       {
-        // Pick a syntax highlighter. 
-        // Can be 'prism' (default), 'shiki' or false to disable any highlighting.
-        syntaxHighlight: 'prism',
-        // If you are using shiki, here you can define a global theme and
-        // add custom languages.
+        // Can be 'shiki' (default), 'prism' or false to disable highlighting
+        syntaxHighlight: 'shiki',
+        // Shiki-specific options
         shikiConfig: {
-          theme: 'github-dark',
-          langs: [],
-          wrap: false,
+          // See built-in themes
+          // https://github.com/shikijs/shiki/blob/main/docs/themes.md#all-themes
+          theme: 'dracula',
+          // Manually specify langs
+          // Note: Shiki has countless langs built-in, including .astro!
+          langs: ['astro'],
+          // Enable word wrap to prevent horizontal scrolling
+          wrap: true,
         },
       },
     ],
   },
 };
 ```
 
-You can read more about custom Shiki [themes](https://github.com/shikijs/shiki/blob/main/docs/themes.md#loading-theme) and [languages](https://github.com/shikijs/shiki/blob/main/docs/languages.md#supporting-your-own-languages-with-shiki).
+Also consider these resources for extended reading:
```

Comment:
We need an explicit warning note SOMEWHERE that if you choose Prism, you also need to add a Prism theme. (Prism will add the classes for you, but requires one of their own themes to define the classes.)

Reply:
Will be adding this into the refactor I mentioned above ☝️